src/devices/lixee.ts

```diff
@@ -1811,7 +1811,7 @@ export const definitions: DefinitionWithExtend[] = [
                 ),
             e
                 .text("tic_command_whitelist", ea.SET)
-                .withDescription("List of TIC commands to be exposed (separated by comma). Reconfigure device after change. Default: all"),
+                .withDescription("List of TIC commands to be exposed (separated by comma). Requires Z2M restart. Default: all"),
```

Comment:
@Koenkk I found that changing the poll interval triggers an update of the list but I think it's better to just suggest a restart.